src/libs/filter-utils.ts

```diff
@@ -32,6 +32,21 @@ export namespace FilterUtilities {
     return true;
   }
 
+
+  /**
+   * Compares severa dates to target dates
+   * 
+   * @param startDate Start date from Severa
+   * @param endDate End date from Severa
+   * @returns 
+   */
+  export const filterByDateSevera = (startDate?: string, endDate?: string): boolean => {
+    if (startDate === null || endDate === null) {
+      return false;
+    }
+    return true;
+  }
```

Comment:
Is this actually doing any filtering by date? I'm also not sure how this is being used in the getWorkHoursHandler but I also left some comments regarding changes therre.